jwql/website/apps/jwql/views.py

```diff
@@ -74,7 +74,8 @@
 from .data_containers import thumbnails_query_ajax
 from .forms import AnomalyQueryForm
 from .forms import FileSearchForm
-from .models import Observation, Proposal, RootFileInfo
+if not os.environ.get("READTHEDOCS"):
```

Comment:
So when we build the docs, we don't want to try and import these modules. Why?

Reply:
Because these modules try to get information from the config file, which is not present on RTD.

Reply:
There was some other similar reason as well. Trying to access something that RTD didn't have access to.

Reply:
Awesome, thanks. I am a novice when it comes to read the docs.